### ISSUE TITLE
"Invite reviewers" URL results in 404

### ISSUE THREAD

**Issue by kannes** [pretalx/pretalx#728]
## Current Behavior

/orga/event/eventname/settings/team/ gives a 404 error:

```

Page not found (404)
Request Method: 	GET
Request URL: 	https://localhost:61006/orga/event/eventname/settings/team/

Using the URLconf defined in pretalx.urls, Django tried these URL patterns, in this order:

    ^orga/ ^login/$ [name='login']
    ^orga/ ^logout/$ [name='logout']
    ^orga/ ^reset/$ [name='auth.reset']
    ^orga/ ^reset/(?P<token>\w+)$ [name='auth.recover']
    ^orga/ ^$
    ^orga/ ^admin/ [name='admin.dashboard']
    ^orga/ ^me$ [name='user.view']
    ^orga/ ^me/subuser$ [name='user.subuser']
    ^orga/ ^invitation/(?P<code>\w+)$ [name='invitation.view']
    ^orga/ ^organiser/$ [name='organiser.list']
    ^orga/ ^organiser/new$ [name='organiser.create']
    ^orga/ ^organiser/(?P<organiser>[a-zA-Z0-9.-]+)/
    ^orga/ ^event/new/$ [name='event.create']
    ^orga/ ^event/typeahead/$ [name='event.typeahead']
    ^orga/ ^event/$ [name='event.list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^$ [name='event.dashboard']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^login/$ [name='event.login']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^reset/$ [name='event.auth.reset']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^reset/(?P<token>\w+)$ [name='event.auth.recover']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^delete$ [name='event.delete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^live$ [name='event.live']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^api/users$ [name='event.user_list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^api/urls/$ [name='url_list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/$ [name='cfp.questions.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/new$ [name='cfp.questions.create']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/remind$ [name='cfp.questions.remind']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/(?P<pk>[0-9]+)/$ [name='cfp.question.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/(?P<pk>[0-9]+)/up$ [name='cfp.questions.up']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/(?P<pk>[0-9]+)/down$ [name='cfp.questions.down']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/(?P<pk>[0-9]+)/delete$ [name='cfp.question.delete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/(?P<pk>[0-9]+)/edit$ [name='cfp.question.edit']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/questions/(?P<pk>[0-9]+)/toggle$ [name='cfp.question.toggle']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/text$ [name='cfp.text.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/types/$ [name='cfp.types.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/types/new$ [name='cfp.types.create']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/types/(?P<pk>[0-9]+)/$ [name='cfp.type.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/types/(?P<pk>[0-9]+)/delete$ [name='cfp.type.delete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/types/(?P<pk>[0-9]+)/default$ [name='cfp.type.default']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/tracks/$ [name='cfp.tracks.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/tracks/new$ [name='cfp.track.create']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/tracks/(?P<pk>[0-9]+)/$ [name='cfp.track.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^cfp/tracks/(?P<pk>[0-9]+)/delete$ [name='cfp.track.delete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^mails/
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^submissions/$ [name='submissions.list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^submissions/new$ [name='submissions.create']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^submissions/cards/$ [name='submissions.cards']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^submissions/feed/$ [name='submissions.feed']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^submissions/statistics/$ [name='submissions.statistics']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^submissions/(?P<code>[\w-]+)/
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^speakers/$ [name='speakers.list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^speakers/(?P<pk>[0-9]+)/$ [name='speakers.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^speakers/(?P<pk>[0-9]+)/reset$ [name='speakers.reset']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^speakers/(?P<pk>[0-9]+)/toggle-arrived$ [name='speakers.arrived']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^info/$ [name='speakers.information.list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^info/new$ [name='speakers.information.create']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^info/(?P<pk>[0-9]+)/$ [name='speakers.information.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^info/(?P<pk>[0-9]+)/delete$ [name='speakers.information.delete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^reviews/$ [name='reviews.dashboard']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/$ [name='settings.event.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/mail$ [name='settings.mail.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/plugins$ [name='settings.plugins.select']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/review/$ [name='settings.review']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/review/phase/(?P<pk>[0-9]+)/up$ [name='settings.review.phase.up']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/review/phase/(?P<pk>[0-9]+)/down$ [name='settings.review.phase.down']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/review/phase/(?P<pk>[0-9]+)/delete$ [name='settings.review.phasedelete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^settings/review/phase/(?P<pk>[0-9]+)/activate$ [name='settings.review.phasedelete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/$ [name='schedule.main']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/import/$ [name='schedule.import']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/export/$ [name='schedule.export']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/export/trigger$ [name='schedule.export.trigger']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/export/download$ [name='schedule.export.download']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/release$ [name='schedule.release']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/quick/(?P<code>\w+)/$ [name='schedule.quick']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/reset$ [name='schedule.reset']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/toggle$ [name='schedule.toggle']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/resend_mails$ [name='schedule.resend_mails']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/rooms/$ [name='schedule.rooms.list']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/rooms/new$ [name='schedule.rooms.create']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/rooms/(?P<pk>[0-9]+)/$ [name='schedule.rooms.view']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/rooms/(?P<pk>[0-9]+)/delete$ [name='schedule.rooms.delete']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/rooms/(?P<pk>[0-9]+)/up$ [name='schedule.rooms.up']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/rooms/(?P<pk>[0-9]+)/down$ [name='schedule.rooms.down']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/api/talks/$ [name='schedule.api.talks']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/api/talks/(?P<pk>[0-9]+)/$ [name='schedule.api.update']
    ^orga/ ^event/(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/api/availabilities/(?P<talkid>[0-9]+)/(?P<roomid>[0-9]+)/$ [name='schedule.api.availabilities']
    ^api/
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/changelog$ [name='schedule.changelog']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/feed.xml$ [name='feed']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/$ [name='schedule']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule.xml$ [name='export.schedule.xml']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule.xcal$ [name='export.schedule.xcal']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule.json$ [name='export.schedule.json']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule.ics$ [name='export.schedule.ics']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/export/(?P<name>[A-Za-z.-]+)$ [name='export']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/v/(?P<version>.+)/$ [name='versioned-schedule']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/v/(?P<version>.+).xml$ [name='versioned-export.schedule.xml']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/v/(?P<version>.+).xcal$ [name='versioned-export.schedule.xcal']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/v/(?P<version>.+).json$ [name='versioned-export.schedule.json']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/v/(?P<version>.+).ics$ [name='versioned-export.schedule.ics']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^schedule/v/(?P<version>.+)/export/(?P<name>[A-Za-z.-]+)$ [name='versioned-export']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^sneak/$ [name='sneak']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^speaker/$ [name='speakers']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^speaker/by-id/(?P<pk>\d+)/$ [name='speaker.redirect']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^talk/$ [name='talks']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^talk/(?P<slug>\w+)/$ [name='talk']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^talk/(?P<slug>\w+)/feedback/$ [name='feedback']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^talk/(?P<slug>\w+).ics$ [name='ical']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^talk/review/(?P<slug>\w+)$ [name='review']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^speaker/(?P<code>\w+)/$ [name='speaker']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^speaker/(?P<code>\w+)/talks.ics$ [name='speaker.talks.ical']
    ^sw.js
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^$ [name='event.landing']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^logout$ [name='event.logout']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^auth/$ [name='event.auth']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^reset$ [name='event.reset']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^login/$ [name='event.login']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^reset/(?P<token>\w+)$ [name='event.recover']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^cfp$ [name='event.start']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^submit/$ [name='event.submit']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^submit/(?P<tmpid>.+)/(?P<step>.+)/$ [name='event.submit']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^invitation/(?P<code>\w+)/(?P<invitation>\w+)$ [name='invitation.view']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^me/$ [name='event.user.view']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^me/delete$ [name='event.user.delete']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^me/submissions/$ [name='event.user.submissions']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^me/mails/$ [name='event.user.mails']
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^me/submissions/(?P<code>[\w-]+)/
    ^(?P<event>[a-zA-Z0-9.-]+)/ ^locale/set [name='locale.set']
    ^control/(?P<path>.*)$ [name='notpretix']
    ^robots.txt$ [name='robots.txt']
    ^$ [name='root.main']
    ^media\/(?P<path>.*)$

The current path, orga/event/eventname/settings/team/, didn't match any of these.

You're seeing this error because you have DEBUG = True in your Django settings file. Change that to False, and Django will display a standard 404 page.

```

## Expected Behavior
Get the page where I can invite reviewers.

## Steps to Reproduce

1. Go to /orga/event/eventname/
2. Hover the loudspeaker icon until the dropdown appears
3. Click the "Invite reviewers" link

## Your Environment

v1.0.0-5-gc02881c

**Comment by rixx** [comment]
Thank you for reporting this issue! The correct place to invite reviewers is in the team settings (Settings -> Teams) and then in your review team(s).

**Comment by kannes** [comment]
Hm, I do not see that option. My Settings only include General, Review and E-mail. Maybe my user is not simply qualified to access the page?

**Comment by rixx** [comment]
Yes, that seems to be the case, then.

**Comment by rixx** [comment]
This issue is fixed in the 1.0.3 bugfix release.
